### ISSUE TITLE
Running an external tool fails if no "apps" are defined in Preferences.

### ISSUE THREAD

**Issue by deathaxe** [FelixBoers/sublime-external-tools#1]
Just tried the package with nothing setup at all but the following in the _ExternalTools.sublime-commands_

```
// ExternalTools Commands - User
[
    {
        "caption": "GitExtensions",
        "command":"external_tools_run",
        "args": { "cmd": [
            "C:\\Program Files (x86)\\GitExtensions\\GitExtensions.exe",
            "$folder"
        ] }
    }
]
```

Calling the "GitExtensions" command fails with:

This is the traceback:
```
Traceback (most recent call last):
  File "C:\Apps\Sublime\sublime_plugin.py", line 916, in run_
    return self.run(**args)
  File "plugin in C:\Apps\Sublime\Data\Installed Packages\ExternalTools.sublime-package", line 13, in run
AttributeError: 'NoneType' object has no attribute 'count'
```

The `get()` should return an empty array by default.

https://github.com/FelixBoers/sublime-external-tools/blob/765ae0dc20e97abd7bf7747ebc8540ea42ce61c5/plugin.py#L11

**Comment by FelixBoers** [comment]
Thanks for your feedback. I've fixed the issue and published a new version 1.0.2.
By the way: If you want to browse your folder path with GitExtensions you have to invoke the `browse` command via command line:

```json
// ExternalTools Commands - User
[
    {
        "caption": "GitExtensions",
        "command":"external_tools_run",
        "args": { "cmd": [
            "C:\\Program Files (x86)\\GitExtensions\\GitExtensions.exe",
            "browse",
            "$folder"
        ] }
    }
]
```

**Comment by deathaxe** [comment]
Thanks for the tip. U used `$file_path` as I have several repos in one "folder"
